### ISSUE TITLE
Authorized? Node doesn't work

### ISSUE THREAD

**Issue by Ruttolo** [guidone/node-red-contrib-chatbot#237]
Hi,

 I started to use redbot just yesterday, creating my first telegram bot and so on….. I'm having some issues with node "authorized" .
I can't restrict my chatbot access to my telegram user (I already put my username as a parameter of telegram receiver). It seems it doesn't evaluate variables using chat functions…….

Thanks,
Alessandro

**Comment by guidone** [comment]
I've update the docs in the `Authorized node`

https://github.com/guidone/node-red-contrib-chatbot/wiki/Authorized-node

**Comment by egorvas** [comment]
Same shit for me(
Already tried to set:

```
    telegram: {
        'Alerts': {
          token: '1081178278:AAESkJ...',
          authorizedUsernames: 'Egorvas,@Egorvas,86733501,egorvas,@egorvas',
          polling: 1000,
          connectMode:'polling',
          parseMode: 'html'
        }
      }
```
And from NodeRed UI as well.
Already tried to set `contextProvider: 'plain-file'`
But nothing work: authorized node says "false" every time and context is empty. If I set authorized context manually it works, but not with authorizedUsernames

**Comment by Ruttolo** [comment]
Egor,

if I remember well, the problem was the username set on telegram’s profile.
Are you sure you are using one of those nicks?

Check “USERNAME” field on Telegram, not “NAME” field.

AR

Inviato da Posta<https://go.microsoft.com/fwlink/?LinkId=550986> per Windows 10

Da: Egor Vasilyev<mailto:notifications@github.com>
Inviato: venerdì 19 giugno 2020 19:04
A: guidone/node-red-contrib-chatbot<mailto:node-red-contrib-chatbot@noreply.github.com>
Cc: Ruttolo<mailto:ilmonco@live.it>; Author<mailto:author@noreply.github.com>
Oggetto: Re: [guidone/node-red-contrib-chatbot] Authorized? Node doesn't work (#237)


Same shit for me(
Already tried to set:

    telegram: {

        'Alerts': {

          token: '1081178278:AAESkJ...',

          authorizedUsernames: 'Egorvas,@Egorvas,86733501,egorvas,@egorvas',

          polling: 1000,

          connectMode:'polling',

          parseMode: 'html'

        }

      }

And from NodeRed UI as well.
Already tried to set contextProvider: 'plain-file'
But nothing work: authorized node says "false" every time and context is empty. If I set authorized context manually it works, but not with authorizedUsernames

**Comment by guidone** [comment]
Thanks @Ruttolo for the answer, I think the problem for @egorvas is that he's not able to make it work using the configuration in settings.js, there was a bug, the parameter was `username` and not `authorizedUsernames`.

It's fixed in `0.19.1`.

I would like to give you and heads up, this is the past, it's still under development but this kind of problem with a plugin of **Mission Control** which will be the control panel to admin the chabot

<img width="1355" alt="Screen Shot on 2020-06-22 at 09-22-57" src="https://user-images.githubusercontent.com/502519/85260046-57c0af00-b46a-11ea-8f7e-41987ba5c62e.png">
